devicon.json

```diff
@@ -6122,6 +6122,26 @@
         "color": "#00b4ff",
         "aliases": []
     },
+    {
+        "name": "qwik",
```

Comment:
The altnames attribute is missing here. If there are no altnames you can just have it be an empty array
```suggestion
        "name": "qwik",
        "altnames": [],
```